src/overlays/actors/ovl_Bg_Jya_Cobra/z_bg_jya_cobra.c

```diff
@@ -26,41 +52,627 @@ const ActorInit Bg_Jya_Cobra_InitVars = {
     (ActorFunc)BgJyaCobra_Update,
     (ActorFunc)BgJyaCobra_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Cobra/func_808958F0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Cobra/func_8089593C.s")
+s16 D_80897308[] = { 0, 0, 0, 0 };
+
+u8 D_80897310[] = { 1, 0, 1, 0 };
+
+s16 D_80897314[] = { -0x4000, 0000, 0x4000, 0000 };
+
+u8 D_8089731C[11][11] = {
+    { 0x00, 0x00, 0x20, 0x80, 0x80, 0x80, 0x80, 0x80, 0x20, 0x00, 0x00 },
+    { 0x00, 0x20, 0x80, 0xA0, 0xA3, 0xA3, 0xA3, 0xA0, 0x80, 0x20, 0x00 },
+    { 0x20, 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80, 0x20 },
+    { 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80 },
+    { 0x80, 0xA3, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA3, 0x80 },
+    { 0x80, 0xA3, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA3, 0x80 },
+    { 0x80, 0xA3, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA3, 0x80 },
+    { 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80 },
+    { 0x20, 0x80, 0xA0, 0xA5, 0xA6, 0xA6, 0xA6, 0xA5, 0xA0, 0x80, 0x20 },
+    { 0x00, 0x20, 0x80, 0xA0, 0xA3, 0xA3, 0xA3, 0xA0, 0x80, 0x20, 0x00 },
+    { 0x00, 0x00, 0x20, 0x80, 0x80, 0x80, 0x80, 0x80, 0x20, 0x00, 0x00 },
+};
+
+u8 D_80897398[3][3] = {
+    { 0x20, 0x80, 0x20 },
+    { 0x80, 0xA0, 0x80 },
+    { 0x20, 0x80, 0x20 },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Cobra/func_808959C4.s")
+Vec3f D_808973A4[] = {
+    { -6.0f, 100.0f, 7.6f },
+    { -12.6f, 69.200005f, -10.0f },
+    { -9.0f, 43.0f, -1.0f },
+    { -3.0f, 15.0f, 8.6f },
+    { -8.6f, 15.0f, 13.5f },
+    { -6.6f, 26.0f, 11.6f },
+    { -12.5f, 43.0f, 8.0f },
+    { -17.2f, 70.0f, 0.6f },
+    { -8.0f, 100.0f, 7.6f },
+    { 6.0f, 100.0f, 7.6f },
+    { 12.6f, 69.200005f, -10.0f },
+    { 9.0f, 43.0f, -1.0f },
+    { 3.0f, 15.0f, 8.6f },
+    { 8.6f, 15.0f, 13.5f },
+    { 6.6f, 26.0f, 11.6f },
+    { 12.5f, 43.0f, 8.0f },
+    { 17.2f, 70.0f, 0.6f },
+    { 8.0f, 100.0f, 7.6f },
+    { 0.0f, 70.0f, -11.3f },
+    { 0.0f, 44.6000023f, -2.0f },
+    { 0.0f, 15.0f, 10.6f },
+    { 0.0f, 15.0f, 0.3f },
+    { 0.0f, 26.0f, 11.6f },
+    { 0.0f, 88.4f, -1.4f },
+    { 0.0f, 95.700005f, 14.900001f },
+    { 0.0f, 101.4f, 5.0f },
+};
+
+Vec3f D_808974DC[] = {
+    { 12.0f, 21.3000011f, -2.5f },  { 30.0f, 21.3000011f, -2.5f }, { -15.0f, 21.3000011f, -2.5f },
+    { -30.0f, 21.3000011f, -2.5f }, { 12.0f, 21.3000011f, -2.5f },
+};
+
+s32 D_80897518[] = { 0x00000080, 0x000000A0, 0x000000A0, 0x00000080 };
```

Comment:
Since these are written to a `u8*`, I think they should be written without the leading zeros.
```suggestion
s32 D_80897518[] = { 0x80, 0xA0, 0xA0, 0x80 };
```